nmkapp/forms.py

```diff
@@ -81,6 +81,46 @@ def clean(self):
         return cleaned_data
 
 
+class SetPasswordForm(forms.Form):
```

Comment:
Gledam sad [SetPasswordForm](https://docs.djangoproject.com/en/1.8/_modules/django/contrib/auth/forms/#SetPasswordForm) i nije mi jasno sta je razlicito? Jedino vidim da je dodat "max_length" i "min_length"? Jel to sve? Ako je to sve, slobodno je izbrisi i koristi djangovu. Nema poente zbog ovoga kopirati ovoliki kod. Ili wrapuj u svoju i promeni ta dva polja.

Mozda sam propustio jos neki deo koji si izmenio? Ako je tako, reci

Reply:
samo sam to dodao, ali ako ostavim djangovu onda ti dozvoli da stavis sifru sa jednim karakterom :/